packages/yoga/src/Tag/native/Tag.jsx

```diff
@@ -1,115 +1,77 @@
 import React from 'react';
 import styled, { withTheme } from 'styled-components';
-import { func, oneOf, oneOfType, bool, node } from 'prop-types';
-import { hexToRgb } from '@gympass/yoga-common';
-import { TriangleAlert } from '@gympass/yoga-icons';
+import { oneOf, node } from 'prop-types';
+import Text from '../../Text';
 
-import { Text } from '../..';
+export const StyledTag = styled.View`
+  justify-content: center;
+  align-items: center;
 
-const StyledTag = styled.View`
   ${({
-    color,
-    full,
+    variant,
     theme: {
       yoga: {
+        colors: {
+          text,
+          elements,
+          feedback: {
+            [variant]: color = { dark: text.secondary },
+            [variant]: borderColor = { dark: elements.selectionAndIcons },
```

Comment:
We have a pattern defined to this type of destruct? At some places we are using [variant]: { light: = text.secondary } and others like this line, if we don't have a pattern defined we could just choose one and use it on all Tag components.